docs/FAQ.md

```diff
@@ -470,3 +471,24 @@ later) finishes before the slow task that triggered speculation. If the speculat
 finishes first then that's good, it is working as intended. If many tasks are speculating, but the
 original task always finishes first then this is a pure loss, the speculation is adding load to
 the Spark cluster with no benefit.
+
+### Why is my query in GPU mode slower than CPU mode?
+
+Below are some troubleshooting tips on GPU query performance issue:
+* Make sure the query in GPU mode is fully on GPU. Please refer to
+  [Getting Started on Spark workload qualification](./get-started/getting-started-workload-qualification.md)
```

Comment:
I think this is a bit too generic.. It the operator is only 2 ms for instance we may not care that everything is fully on GPU.  Also do we want to point to qualification tool or prefer the profiling tool and/or specifically look at the plan

Reply:
that seems fine